go.mod

```diff
@@ -20,5 +20,5 @@ require (
 	golang.org/x/sync v0.0.0-20181108010431-42b317875d0f // indirect
 	golang.org/x/sys v0.0.0-20181122145206-62eef0e2fa9b // indirect
 	google.golang.org/genproto v0.0.0-20181109154231-b5d43981345b // indirect
```

Comment:
Actually, this may not be a complete fix. This indirect required version of `genproto` requires grpc v1.16.0:

https://github.com/google/go-genproto/commit/b5d43981345b#diff-37aff102a57d3d7b797f152915a6dc16R6

I'm not sure if it'll continue to cause a problem. /cc @bcmills @thepudds

Reply:
`genproto` isn't used directly in this module, so to be on the safer side, I'll also drop this indirect module requirement. That completely gets rid of grpc@v1.16.0.

Reply:
@dmitshur does `go mod tidy` drop `genproto` for you automatically after you change the `go.mod` to require grpc `v1.17.0` instead of `v1.16.0`? Or are you saying dropping `genproto` would be a manual edit?

Reply:
It has to be an explicit manual edit, because it's an indirect module requirement.

Reply:
OK, looks removing `genproto` from the `go.mod` here would in effect downgrade `genproto` here from a November 2018 commit (20181109154231) to an August 2018 commit (20180817151627).